floodviz/templates/index.html

```diff
@@ -11,9 +11,39 @@
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/peakflow.css') }}" />
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}" />
 	<title>Flood Viz</title>
-
 	<script type="application/ld+json">
 		{{ linked_data | tojson | safe }}
+
+	</script>
+	<script type="text/javascript">
+		(function (i, s, o, g, r, a, m) {
+			i['GoogleAnalyticsObject'] = r;
+			i[r] = i[r] || function () {
+					(i[r].q = i[r].q || []).push(arguments)
+				}, i[r].l = 1 * new Date();
+			a = s.createElement(o),
+				m = s.getElementsByTagName(o)[0];
+			a.async = 1;
+			a.src = g;
+			m.parentNode.insertBefore(a, m)
+		})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+		ga('create', 'UA-103831764-1', 'auto');
+		ga('send', 'pageview');
+
+		// Global Namespace for project - FV
+		var FV = FV || {};
+
```

Comment:
This needed to be defined up here so we have the ability to call FV.ga_send_event on html content (links, icons, etc)